File: src/experiment/CheeseGrater/src/main/java/sdu/pizza/cheesegrater/PizzaDoneMessage.java
package sdu.pizza.cheesegrater;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class PizzaDoneMessage {
    @JsonProperty("pizzaId")
    private int pizzaId;

    @JsonProperty("orderId")
    private int orderId;

    @JsonProperty("doneMsg")
    private boolean doneMsg = true;
}

File: src/experiment/cheese/CheeseGrater/Data/PizzaDoneMessage.cs
using System.Text.Json.Serialization;

namespace CheeseGrater.Data;

public class PizzaDoneMessage
{
    [JsonPropertyName("pizzaId")]
    public int PizzaId { get; set; }
    
    [JsonPropertyName("orderId")]
    public string OrderId { get; set; }
    
    [JsonPropertyName("doneMsg")]
    public bool DoneMsg { get; set; } = true;
}